src/groups/mqb/mqbblp/mqbblp_routers.t.cpp

```diff
@@ -519,8 +549,12 @@ int main(int argc, char* argv[])
     // Initialize Crc32c
     bmqp::Crc32c::initialize();
 
+    // bslma::Default::setDefaultAllocator(&ma);
+
     TEST_PROLOG(mwctst::TestHelper::e_DEFAULT);
 
+    // bslma::DefaultAllocatorGuard _defAllocGuard(&ma);
```

Comment:
Changes in this file are not needed, right?

Reply:
Thanks!